src/poojap/Assignment_38.java

```diff
@@ -0,0 +1,37 @@
+/*Assignment - 38 : 11th Dec'2020 
+Verify number is in Fibonacci series or not.*/
+
+package poojap;
+
+import java.util.Scanner;
+
+public class Assignment_38 {
+	void findElementInFibonacciSeries(int num) {
+		int first = 0, second = 1;
+		System.out.println("Fibonacci Series:");
+		System.out.print(first + "," + second);
+		boolean flag = false;
```

Comment:
Refer below for handle boundary value

https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/1267/files